### ISSUE TITLE
Browse an office and Postal code lookup

### ISSUE THREAD

**Issue by shawnthompson** [ServiceCanada/GCWeb#3]
[Browse an office and Postal code lookup](http://servicecanada.github.io/GCWeb/contact/find-en.html)

Removed the Heading above the tabs panel because it repeated the main heading on the page.

Left the "Find an office by postal code" for later because it will take me longer to markup with the interactive map. Might need help with that from @masterbee

**Comment by DK-TBS** [comment]
Agreed on removing that extra heading, I don't know why I had left that in there.  I'll update the spec/wireframes to match what you've done.

* Everything in the “Browse office locations” tab looks good.
* I'll hold off on reviewing the "Find an office by postal code" stuff until you're finished with it.

**Comment by shawnthompson** [comment]
@DK-TBS @pcwsmith I pulled the map out of the tab for now.

http://servicecanada.github.io/GCWeb/contact/region-en.html

**Comment by DK-TBS** [comment]
Safe to put this whole page/template on hold.  Seems less and less likely that it's going to end up being put to use.
